ts/Core/Chart/ChartDefaults.ts

```diff
@@ -492,7 +492,7 @@ const ChartDefaults: ChartOptions = {
      *
      * @type       {string}
      * @default    {highcharts} undefined
-     * @default    {highstock} x
+     * @default    {highstock} undefined
```

Comment:
I've removed the default value for this deprecated option so that the setZoomOptions function is much shorter. Now, chart.options.zoomType, chart.options.pinchType is set only when the users set it (it is not always taken from ChartDefaults).